Parallel_Fiji_Visualizer/src/main/java/edu/abc/berkeley/PWZGUI.java

```diff
@@ -348,36 +383,28 @@ private void loadfile(String filepath){
 
         if(cImagePlus == null) return;
 
-        ImageStack cImageStack = cImagePlus.getImageStack();
-        Object[] cImageObj = cImageStack.getImageArray();
+		// pwzc.parallelWriteZarr(this.filepath, this.cImageObj, this.startX, this.startY, this.startZ, this.endX, this.endY, this.endZ, this.chunkSizeX, this.chunkSizeY, this.chunkSizeZ, 1, this.compressor, 1, this.bits);
+    }
+
+    private boolean checkExtension(String path){
+        String extension = "";
+        int i = path.lastIndexOf('.');
 
+        if(i != -1) extension = path.substring(i+1);
+
+        // System.out.println("LOGGER: " + extension + ", i = " + i); // For debugging purposes.
         
-		int bits = cImageStack.getBitDepth(); // May need to get bits another way
-		String fileName = file.getPath();
-		int x = cImageStack.getWidth();
-		int y = cImageStack.getHeight();
-		int z = cImageStack.getSize();
+        if(extension.equals("zarr")) return true;
 
-		// pwzc.parallelWriteZarr(fileName, cImageObj, 0, 0, 0, y, x, z, 256, 256, 256, 1, "lz4", 1, bits);
+        return false;
     }
 
-    
-    private void checkboxClicked(){ System.out.println("[DEBUGGING]: Check Box Clicked!"); } // Does smthing when this check box is clicked.
-
-    // When "Save Changes" button is clicked, the given inputted information is updated
-    // Little redundant, for now was trying to get interface working with updating the JTable.
+    // When "Submit changes" button is clicked, the given inputted information is updated
     private void updateTable(){
```

Comment:
This function should call my function but I can write that in myself when I pull this since I'll be testing it.